### PR TITLE
IVRCallReportsTestUiTest

### PR DESCRIPTION
**Pull request by PiotrZurawskiSolDevelo**

### CODE REVIEW

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by MikolajBasSoldevelo** [comment]
test this please

**Comment by MikolajBasSoldevelo** [comment]
test this please

File: ebodac/src/test/java/org/motechproject/ebodac/uitest/page/ReportPage.java [hunk 1]
```diff
@@ -15,6 +17,38 @@
 
     static final By FOLLOW_UPS_AFTER_PRIME_INJECTION_REPORT = By.linkText("Follow-ups After Prime Injection Report");
 
+    static final By CALL_DETAIL_REPORT = By.linkText("Call Detail Record");
+
+    //TABLE
+    static final String CALL_DETAIL_REPORT_TABLE_ID="jqgh_instancesTable_id";
```
Comment (pmuchowski) [inline ebodac/src/test/java/org/motechproject/ebodac/uitest/page/ReportPage.java:23]: add white spaces before and after =

File: ebodac/src/test/java/org/motechproject/ebodac/uitest/page/ReportPage.java [hunk 2]
```diff
@@ -31,6 +65,41 @@ public void showFollowUpsAfterPrimeInjectionReport () throws InterruptedExceptio
...
 
+    public void showCallDetailRecord() throws InterruptedException{
+        clickWhenVisible(CALL_DETAIL_REPORT);
+    }
+
+    public boolean checkIfTableOfCallDetailRecordInstancesIsVisible() throws InterruptedException {
+        boolean result=true;
+        String name[] = new String[14];
+
+        name[0]=String.valueOf(CALL_DETAIL_REPORT_TABLE_ID);
```
Comment (pmuchowski) [inline ebodac/src/test/java/org/motechproject/ebodac/uitest/page/ReportPage.java:76]: add white spaces before and after =

File: ebodac/src/test/java/org/motechproject/ebodac/uitest/page/ReportPage.java [hunk 3]
```diff
@@ -31,6 +65,41 @@ public void showFollowUpsAfterPrimeInjectionReport () throws InterruptedExceptio
...
+        name[6]=String.valueOf(CALL_DETAIL_REPORT_TABLE_TEMPLATE_NAME);
+        name[7]=String.valueOf(CALL_DETAIL_REPORT_TABLE_PROVIDER_EXTRA_DATA);
+        name[8]=String.valueOf(CALL_DETAIL_REPORT_TABLE_MOTECH_CALL_ID);
+        name[9]=String.valueOf(CALL_DETAIL_REPORT_TABLE_PROVIDER_CALL_ID);
+        name[10]=String.valueOf(CALL_DETAIL_REPORT_TABLE_MOTECH_TIMESTAMP);
+        name[11]=String.valueOf(CALL_DETAIL_REPORT_TABLE_PROVIDER_TIMESTAMP);
+        name[12]=String.valueOf(CALL_DETAIL_REPORT_TABLE_CALL_DURATION);
+        name[13]=String.valueOf(CALL_DETAIL_REPORT_TABLE_MESSAGE_PERCENT_LISTENED);
+
+        for(int i=0;i<14;i++)
```
Comment (pmuchowski) [inline ebodac/src/test/java/org/motechproject/ebodac/uitest/page/ReportPage.java:91]: the same here also before and after < and after ; and before (

**Comment by kkaczmarczyk** [comment]
test this please

**Pull request closed without merging**